doc/cdi-operator-olm.md

```diff
@@ -35,46 +35,64 @@ DOCKER_REPO=<repo> DOCKER_TAG=<docker tag> PULL_POLICY=<pull policy> VERBOSITY=<
 QUAY_NAMESPACE=<quay namespace> QUAY_REPOSITORY=<quay repo> QUAY_USERNAME=<quay username> QUAY_PASSWORD=<quay password> CSV_VERSION=<csv version > make olm-push
 ```
 #### Install OLM and Marketplace operators on cluster
-- Install OLM operator from cloned operator-lifecycle-manager repo 
+- Install OLM operator from cloned operator-lifecycle-manager repo and wait untill all pods are Running and Ready. 
 
 ```bash
 kubectl apply -f $GOPATH/src/github.com/operator-framework/operator-lifecycle-manager/deploy/upstream/quickstart/olm.yaml
 ```
-- Install marketplace operator from cloned operator-marketplace repo
+- Install marketplace operator from cloned operator-marketplace repo and wait untill all pods are Running and Ready.
 ```bash
 kubectl apply -f $GOPATH/src/github.com/operator-framework/operator-marketplace/deploy/upstream/ --validate=false
 ```
-###CDI installation by means of OLM and Marketplace operators
+- Wait till cdi marketplace is Running and Ready.
```

Comment:
Wait until CDI market place operator is Running and Ready

Reply:
Done